static/templates/settings/bot-settings.handlebars

```diff
@@ -31,7 +31,7 @@
             <label for="bot_short_name">{{t "Username" }}</label>
             <input type="text" name="bot_short_name" id="create_bot_short_name" class="required bot_local_part"
                    placeholder="bot_user_name" value="" />
```

Comment:
Just noticed this -- should we change `bot_user_name` here to be a more reasonable placeholder?